bugbear.py

```diff
@@ -653,6 +655,7 @@ def visit(self, node):
     )
 )
 B006.mutable_literals = (ast.Dict, ast.List, ast.Set)
+B006.mutable_comprehensions = (ast.ListComp, ast.DictComp)
```

Comment:
I think set comprehensions (`{x for x in y}`) is also suitable in this context
```suggestion
B006.mutable_comprehensions = (ast.ListComp, ast.DictComp, ast.SetComp)
```

Reply:
And I knew I was forgetting something, thanks for noticing I'll fix it soon once I get back to my computer at home.